### PR TITLE
Migrate NotificationBar.js to styled components

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/3289

# Screenshots

<details>
<summary>Desktop</summary>

image
</details>

<details>
<summary>Mobile</summary>

Screen Shot 2020-11-15 at 22 16 16
</details>

### CODE REVIEW

File: components/NotificationBar.js [hunk 1]
```diff
@@ -1,12 +1,63 @@
...
 
 import AcceptRejectButtons from './host-dashboard/AcceptRejectButtons';
+import Container from './Container';
 import { Flex } from './Grid';
+import { H1, P } from './Text';
 
 const logo = '/static/images/opencollective-icon.svg';
 
+const NotificationBarContainer = styled.div`
+    .oc-message .logo {
```
Comment (Betree) [inline components/NotificationBar.js:14]: We should ideally get rid of these CSS classes, use dedicated styled components where we can.

File: components/NotificationBar.js [hunk 2]
```diff
@@ -1,12 +1,63 @@
...
+      margin: 10px;
+    }
+
+    .error .oc-message {
+      position: fixed;className="oc-message"
```
Comment (Betree) [inline components/NotificationBar.js:19]: Seems there's a problem here

**Review by Betree — CHANGES_REQUESTED** [review]

File: components/NotificationBar.js (hunk 2 above)
Comment (SudharakaP) [reply components/NotificationBar.js]: Thanks. That's some unintended pasting on my part. Just removed it. 👍🏽

File: components/NotificationBar.js (hunk 1 above)
Comment (SudharakaP) [reply components/NotificationBar.js]: @Betree : I've refactored to get rid of the CSS classes as much as possible. Take a look and let me know if there's anything else to be done. 😄

**Comment by Betree** [comment]
All good!

**Pull request merged**